images/index.html

```diff
@@ -0,0 +1,176 @@
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Beautiful Free Nova template</title>
+    <link rel="stylesheet" href="style/style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap">
+</head>
+<body>
+  <div class="green">
+    <header class="top">
+        <img src="logo.svg" class="logo">
+        <nav>
+            <ul class="menu">
+                <li><a href="#">The Team</a></li>
+                <li>
+                    <a href="#">Pricing</a>
+                </li>
+                <li><a href="#">Features</a></li>
+                <li><a class="button" href="#">DONWLOAD FREE!</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <section class="article">
+        <div class="col">
+            <h1 font-size:36px, font-weight: 600>Beautiful Free Nova template</h1>
+            <p class="header-text"> A top notch premium quality template for your next web project.</p>
+            <h4> Download FREE!</h4>
+        </div>
+        <div class="col">
+            <img src="screen.png" class="ipad-screen">
+        </div>
+    </section>
+
+  </div> <!-- ten zielony -->
+ <section class="black-section">
+    <h1> FEATURES </h1>
+    <div class="container2">
+        <div class="feature">
+            <img src="responsive.svg" alt="Icon 1" class="icon">
+            <h3>FULLY RESPONSIVE</h3>
+            <p>Looks amazing on any device: smartphone, tablet, laptop and desktop.</p>
+        </div>
+        <div class="feature">
+            <img src="customizable.svg" alt="Icon 2" class="icon">
+            <h3>CUSTOMIZABLE</h3>
+            <p>Change the colors, pictures or any of the sections to suit your needs.</p>
+        </div>
+        <div class="feature">
+            <img src="design.svg" alt="Icon 3" class="icon">
+            <h3>SLICK AND BEAUTIFUL DESIGN</h3>
+            <p>Trendy and fresh design, fits any website.</p>
+        </div>
+    </div>
+    </section>
+
+ <section class="grey-section">
+    <img src="
+    logos.png">
+ </section>
+ <section class="white-section">
+<div class="col2">
+    <img src="imac.png" class="imac">
+</div>
+<div class="col2">
+    <h2 class="simple-and-beautiful">SIMPLE AND BEAUTIFUL</h2>
+    <div class="text-nova-theme"><p>Use Nova theme for your next web project. It is <br>
+        completely customizable so you can change any of <br>
+        the sections to fit your needs. Nova Theme is Free for <br>
+        any kind of use, personal and commercial. Have fun <br>
+        and good luck!</p></div>
+</div>
+ </section>
+
+<section class="grey-section">
+    <header>
+        <div>
+            <h3 class="choose-your-plan">CHOOSE YOUR PRICING PLAN</h3>
+        </div>
+    </header>
+    <section>
+        <div>
+            <h5 class="pricing-plans">Pick any of our super affordable pricing plans</h5>
+        </div>
+        <section class="pricing">
+        <div class="pricing-plan">
```

Comment:
Uwaga na formatowanie. 

PS. Ponieważ mamy "listę cen" to tutaj również można zastosować `ul > li` (każdy pakiet cenowy) zamiast. `section > div`